basic/exercise1/script.js

```diff
@@ -1,28 +1,19 @@
 'use strict';
 function formatDate(date) {
+    function insertingZero(i) {
```

Comment:
Please rename:
- function name - it should specify what the function does or what might be the result of the function. insertingZero is not a good name as it refers to continuous action. Here it is not a continuous action. It is better to call it, lets say formatAsTwoDigits
- parameter - is "i" name descriptive here? No. It should be more meaningful, like for example number.